tensorly/contrib/sparse/backend/tensorflow_backend.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import tensorflow as tf
+from ....backend.core import Backend
+from copy import copy as _py_copy
+
+
+def is_sparse(x):
+    return isinstance(x, tf.sparse.SparseTensor)
+
+
+class TensorflowSparseBackend(Backend):
+    backend_name = 'tensorflow'
```

Comment:
Should the backend_name be `tensorflow` or `tensorflow.sparse`? In [`numpy_backend`](https://github.com/tensorly/tensorly/blob/b896b142b09f559f981237d785738467a05dcaf3/tensorly/contrib/sparse/backend/numpy_backend.py#L25) the name is `numpy.sparse`, but if I set the name to `tensorflow.sparse` here, I get a `ValueError: Unknown backend name 'tensorflow.sparse', known backends are ['numpy', 'tensorflow']` error.